packages/core/src/model.ts

```diff
@@ -198,13 +198,12 @@ export class Model implements Storage {
   /**
    * Construct a new proper document with all desired fields.
    * @param _class
-   * @param _id
    * @param layout
    */
-  public newDoc<T extends Doc> (_class: Ref<Class<T>>, _id: Ref<Doc>, layout: AnyLayout): T {
-    const doc = (this.assign({}, _class, layout) as unknown) as T
-    doc._id = _id
-    return doc
+  public createDocument<T extends Doc> (_class: Ref<Class<T>>, layout: Partial<T>): T {
+    const doc = this.assign({}, _class, layout as unknown as AnyLayout)
+    doc._id = generateId()
+    return doc as unknown as T
```

Comment:
In the past `createDocument` was accepting optional `id` param. Is this not a case now?

Reply:
It is used only in model generation in few cases, will revert this change, thanks.

Reply:
reverted.